files/templates/additional-scrape-configs.template.yaml

```diff
@@ -2482,39 +2482,4 @@
   # Add installation label.
   - target_label: installation
     replacement: {{ .Installation }}
-# prometheus
```

Comment:
Why delete this?

Reply:
Because I needed to create a service for the ingress and added the annotations to the service so this is getting picked up by the managed apps part

Reply:
Got it, thanks.